packages/strapi-plugin-upload/admin/src/components/CardPreview/index.js

```diff
@@ -11,16 +11,16 @@ import Image from './Image';
 const CardPreview = ({ hasError, url, type }) => {
   const isFile = getType(type) === 'file';
 
-  const renderFile = () => <FileIcon fileType={type} />;
-
-  const renderImage = () => <Image src={url} />;
+  if (hasError) {
+    return (
+      <Wrapper isFile>
```

Comment:
is it always a file in this case?

Reply:
It's currently called a file because the icon is a broken file (when the user wasn't able to upload it) and the UI is exactly like it's a file and nothing else. Do you think it can be called anything else?